docs/dev/how-to/send-transaction-l1-l2.md

```diff
@@ -233,7 +233,31 @@ Along with zkSync Era's built-in censorship resistance that requires multi-layer
 
 User needs to perform next steps:
 
-1. Run local node dockerized containers. [`Instructions how to run it`](https://github.com/matter-labs/local-setup/tree/main)
+1. Run local node dockerized containers. [`Instructions how to run it`](https://github.com/matter-labs/local-setup/tree/main) or use [`zksync-cli`](https://github.com/matter-labs/zksync-cli):
+
+::: code-tabs
+@tab npx
+
+```npx
+  npx zksync-cli@latest dev config
+  // choose: Dockerized node - Persistent state, includes L1 and L2 nodes
+  // choose: BE and Portal (optional)
+  npx zksync-cli@latest dev start
```

Comment:
`npx` will automatically fetch the latest version of the CLI like so:

```bash
npx zksync-cli dev config
Need to install the following packages:
zksync-cli@0.11.0
Ok to proceed? (y) y
? Node type to use Dockerized node
? Additional modules to use

Configuration saved successfully!
Start configured environment with `npx zksync-cli dev start`
```

```suggestion
  npx zksync-cli dev config
  // choose: Dockerized node - Persistent state, includes L1 and L2 nodes
  // choose: BE and Portal (optional)
  npx zksync-cli dev start
```

Reply:
I guess this works if you didn't use previous versions.
For me it was 0.5 version and it tried to use 0.5 versions all the times.
So only this tag helped me.